algrtm/src/components/StoryComponent.js

```diff
@@ -0,0 +1,136 @@
+              className="h-full w-1 bg-gray-200 absolute left-1/2 transform -translate-x-1/2"
+              style={{ height: "100%", top: "0" }}
+            ></div>
+            <div
+              className={`flex ${index % 2 === 0 ? "" : "flex-row-reverse"}`}
+            >
+              <div
+                className={`flex-shrink-0 w-10 h-10 items-center justify-center inline-flex absolute z-10 left-1/2 transform -translate-x-1/2`}
+              >
+                <svg xmlns="http://www.w3.org/2000/svg" width="8" height="8">
+                  <rect width="8" height="8" fill="#B3B3B3" />
+                </svg>
+              </div>
+              <div
+                className={`flex-grow max-w-[50%] ${
+                  index % 2 === 0 ? "pr-[4.875rem]" : "pl-[4.875rem]"
+                }`}
+              >
+                <p className="p1SB text-blueLight-blue50">{step.step}</p>
+                <h4 className="h4 text-neutralLight-neutral10 tracking-wider">
+                  {step.title}
+                </h4>
+                <h6 className="h6 text-neutralLight-neutral20">
+                  {step.description}
+                </h6>
+              </div>
+            </div>
+          </div>
+        ))}
+      </div>
+
+      <div className="flex gap-[1.5rem] px-[12.62rem] pt-[5.88rem] justify-center items-center">
+        <div className="rounded-md bg-neutralLight-neutral90 py-[4rem] px-[1.5rem] flex flex-col items-center gap-[2.75rem] w-full max-w-[31.625rem]">
+          <div className="text-[3.25rem] font-semibold text-center">
+            <p className="text-neutralLight-neutral10">
+              Stay up to date <br />
+              with <span className="text-blueLight-blue50">PianoRoll</span>
+            </p>
+          </div>
+          <div className="flex items-center justify-center gap-[1.5rem]">
+            {socialIconsArray.slice(0, 3).map((image) => {
+              return (
+                <Link
+                  key={image.name}
+                  href={externalLinks[image.name]}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <Image
+                    src={`/assets/social/${image.name
+                      .toLowerCase()
+                      .split(" ")
+                      .join("-")}-logo.svg`}
+                    width={image.size.width}
+                    height={image.size.height}
+                    alt={`${
+                      image.name.charAt(0).toUpperCase() + image.name.slice(1)
+                    } logo`}
+                    className="transition-transform duration-300 transform hover:scale-125"
+                  />
+                </Link>
+              );
+            })}
+          </div>
+        </div>
+        <div className="rounded-md bg-neutralLight-neutral100 py-[4rem] px-[1.5rem] flex flex-col items-center gap-[2.75rem] border-1 border-neutralLight-neutral90 w-full max-w-[31.625rem]">
```

Comment:
A small mistake here, `border-1` does not exist in Tailwind, so the border is missing here, instead it is just `border`. In addition, the radius should be 16px